src/parser/monk/mistweaver/modules/spells/azeritetraits/FontOfLife.js

```diff
@@ -55,16 +56,24 @@ class FontOfLife extends Analyzer {
 
   statistic() {
     return (
-      <TraitStatisticBox
-        position={STATISTIC_ORDER.OPTIONAL()}
-        trait={SPELLS.FONT_OF_LIFE.id}
-        value={(
-          <>
-            {formatPercentage(this.healing / this.getAbility(SPELLS.ESSENCE_FONT_BUFF.id).healingEffective)} % of Essence Font Healing
-          </>
-        )}
-        tooltip={`Added a total of ${formatNumber(this.healing)} to your Essence Font Bolts.`}
-      />
+      <AzeritePowerStatistic
+        size="flexible"
+        tooltip={(<>Added a total of ${formatNumber(this.healing)} to your Essence Font Bolts.</>)}
```

Comment:
Since you're not using any JSX inside, you can use simple string here
```suggestion
        tooltip={`Added a total of ${formatNumber(this.healing)} to your Essence Font Bolts.`}
```